### ISSUE TITLE
Many warnings if "Debug information Format" of Xcode project settings is "DWARF with dSYM file".

### ISSUE THREAD

**Issue by kiyotakagoto** [AgoraIO/Basic-Audio-Call#1]
As reported on [a issue](https://github.com/AgoraLab/AgoraDemo/issues/3), 
so many warnings come out, like:
```
warning: (x86_64) /Users/yurun/.jenkins/workspace/agora-release-ios/media_sdk2/proj.ios/output/tmp/mediasdk.build/Release-iphonesimulator/mediasdk.build/Objects-normal/x86_64/json_wrapper.o unable to open object file: No such file or directory.
```

It seems that this issue occurs if "Debug information Format" of Xcode project settings is "DWARF with dSYM file".

**Comment by PRWrl** [comment]
Hi @kiyotakagoto we are working on this warning and plan to fix it in next SDK version.

**Comment by PRWrl** [comment]
The newly released version 2.4.0 has resolved this issue, you can download from [here](https://docs.agora.io/en/Agora%20Platform/downloads).

**Comment by kiyotakagoto** [comment]
thanks!
